packages/peregrine/lib/Router/magentoRouteHandler.js

```diff
@@ -94,8 +94,12 @@ export default class MagentoRouteHandler extends Component {
                 route: pathname
             });
 
-            const { type, id } = resolvedRoute;
+            // urlResolver query returns null if a route can't be found
+            if (resolvedRoute === null) {
```

Comment:
This seems to rely on a literal null, and I'm not sure that's guaranteed. Any reason not to just test it for truthiness here?

Reply:
In all of my tests null was returned, if there was no match, that's why I decided to make it rely on literal null. And I wanted to have these changes as unobstrusive as possible. But I'm also fine with just checking for truthiness. Should I change this?

Reply:
Yeah, would you mind?
```suggestion
            if (!resolvedRoute) {
```

Reply:
@zetlen Applied your suggested change